### PR TITLE
Add kind to the base builder used by circle

### PR DESCRIPTION
**Pull request by costinm**

### CODE REVIEW

**Comment by sdake** [comment]
@costinm pr lgtm, however, as you can see the gate is unhappy :). Has the circleci image been pushed to the hub?


Cheers
-steve

**Comment by howardjohn** [comment]
@costinm there have been a lot of changes around this area lately, is this still needed?

File: .circleci/Makefile [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
+CI_VERSION ?= 2019-05-09
+CI_HUB ?= istionightly
 
 ci.image:
 	docker build -t "${CI_HUB}/ci:$(CI_VERSION)" -f Dockerfile .
 
 ci.push:
 	docker push --config=/tmp/circle "${CI_HUB}/ci:$(CI_VERSION)" 
 
+docker.push:
```
Comment (howardjohn) [inline .circleci/Makefile]: isn't this the same as `ci.push`?

**Comment by howardjohn** [comment]
Do we still need this now that we have moved most things to prow @costinm

**Pull request closed without merging**